smart-contracts/contracts/cl-vault/Cargo.toml

```diff
@@ -49,7 +49,7 @@ cw-dex = "0.1.1"
 cw-dex-router = {version = "0.1.0", features = ["library"]}
 cw-storage-plus = "1.0.1"
 cw-utils = "0.16.0"
-cw-vault-multi-standard = {git = "https://github.com/quasar-finance/cw-vault-standard", features = ["lockup", "force-unlock"]}
+cw-vault-multi-standard = {git = "https://github.com/quasar-finance/cw-vault-standard", features = ["lockup", "force-unlock"], branch = "feat/change-convert-assets-return-type"}
```

Comment:
Cargo build fails with error,
error: failed to select a version for the requirement `osmosis-test-tube = "^17.0.0-rc0"`
candidate versions found which didn't match: 22.0.0
I think this version 17.0.0-rc0 does not exist any more